05-geometry-operations.Rmd

```diff
@@ -375,8 +375,8 @@ op = par(mar = rep(0, 4))
 x = b[1]
 y = b[2]
 x_and_y = st_intersection(x, y)
-plot(b, border = "grey")
-plot(x_and_y, col = "lightgrey", border = "grey", add = TRUE) # intersecting area
+plot(b, border = "gray")
+plot(x_and_y, col = "lightgray", border = "gray", add = TRUE) # intersecting area
```

Comment:
I thought there was an issue but testing suggests it's OK:

``` r
plot(1:9, col = "gray")
```

![](https://i.imgur.com/bHA0PGz.png)<!-- -->

<sup>Created on 2024-09-28 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

Reply:
Do we now use "gray" throughout?